tests/ext/test_commandhandler.py

```diff
@@ -181,7 +181,11 @@ async def test_basic(self, app, command):
 
     @pytest.mark.parametrize(
         "cmd",
-        ["way_too_longcommand1234567yes_way_toooooooLong", "ïñválídletters", "invalid #&* chars"],
+        [
+            "way_too_longcommand1234567yes_way_toooooooLong",
+            "ïñválídletters",
+            "invalid #&* chars",
+        ],
```

Comment:
Is this change related to the PR? If not, please revert

Reply:
Reverted